### PR TITLE
Implement MITRE D3FEND matrix #975

### PR DESCRIPTION
**Pull request by cvandeplas**
See issue #975  for screenshots and all. 

PR for unit testing as we all know I often break things :-D

### CODE REVIEW

**Comment by cvandeplas** [comment]
@adulau ready for review. 
I also changed the links of README to the misp-galaxy.org website as this is a much cleaner look.
Would you be able to update the website again please? 

Note that the relations do not follow the [misp-objects relation definitions](https://github.com/MISP/misp-objects/blob/main/relationships/definition.json).

**Review by adulau — APPROVED** [review]
I love the idea!

**Review by adulau — APPROVED** [review]
Good point to change to the misp-galaxy page.

File: tools/gen_mitre_d3fend.py [hunk 1]
```diff
@@ -7,22 +25,24 @@
 d3fend_url = 'https://d3fend.mitre.org/ontologies/d3fend.json'
 d3fend_full_mappings_url = 'https://d3fend.mitre.org/api/ontology/inference/d3fend-full-mappings.json'
 
+# we love eating lots of memory
```
Comment (adulau) [inline tools/gen_mitre_d3fend.py:28]: BMI approved

**Comment by adulau** [comment]
Thanks a lot, it's a great addition. Maybe we should notify MITRE about it.

**Pull request merged**